CMSgov/design-system
# [NO-TICKET] idle timeout button variation update

## Summary
Per stakeholder request, updating the 'end session' button to be the default button styling instead of the transparent treatment

## How to test

- `yarn storybook`
- Navigate to Idle Timeout > Idle Timeout Dialog to see change

Hgov:
<img width="576" alt="Screen Shot 2022-02-28 at 11 15 08 AM" src="https://user-images.githubusercontent.com/33579665/156043167-049639d1-d859-4d1f-9d03-8aea9d2c8704.png">


Core
<img width="576" alt="Screen Shot 2022-02-28 at 11 13 44 AM" src="https://user-images.githubusercontent.com/33579665/156043171-99aef6c3-9fc8-493c-be83-f409cf98598a.png">
